server/blob/blob-objectstorage/src/main/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAO.java

```diff
@@ -153,8 +153,8 @@ public BucketName getDefaultBucketName() {
         return defaultBucketName;
     }
 
-    public void deleteContainer() {
-        blobStore.deleteContainer(defaultBucketName.asString());
+    public void deleteContainer(BucketName bucketName) {
```

Comment:
This is `deleteBucket`, no?

Reply:
Yes should be `Mono<Void> delete(BucketName bucketName)`